metashare/api/jobs.py

```diff
@@ -184,17 +195,88 @@ def create_branches_on_github_then_create_scratch_org(
             repo_branch=commit_ish,
             project_path=repo_root,
         )
+        get_unsaved_changes(scratch_org, should_save=False)
```

Comment:
@wlonk Before this runs (probably above in `create_org_and_run_flow`?), it needs to store the _initial_ set of `latest_revision_numbers` on the org. Otherwise the first time `get_unsaved_changes` is called here, it returns _all_ revisions as if they're unsaved.

Reply:
That's by design; the intent is to set `latest_revision_numbers` with whatever was set up by the flow at org-creation. But that does also mark it as having unsaved changes, doesn't it?

Reply:
Right; we need to separate those two for the initial run.